axum-extra/src/response/erased_json.rs

```diff
@@ -28,6 +28,7 @@ use serde::Serialize;
 ///     }
 /// }
 /// ```
+#[cfg_attr(docsrs, doc(cfg(feature = "erased-json")))]
```

Comment:
For future reference, this is needed because of https://github.com/rust-lang/rust/issues/83428.